README.md

```diff
@@ -486,6 +486,7 @@ Services to securely store your Docker images.
 * [Rapid Dashboard](https://github.com/ozlerhakan/rapid) - A simple query dashboard to use Docker Remote API by [@ozlerhakan](https://github.com/ozlerhakan/)
 
 # Useful Resources
+* [Awesome Kubernetes](https://github.com/ramitsurana/awesome-kubernetes) by [@ramitsurana](https://github.com/ramitsurana)
```

Comment:
What did you mean by "could you add it to the bottom of the readme as well please"?